test/Microsoft.ML.Benchmarks/Harness/Configs.cs

```diff
@@ -18,19 +18,18 @@ public RecommendedConfig()
                 .With(CreateToolchain())); // toolchain is responsible for generating, building and running dedicated executable per benchmark
 
             Add(new ExtraMetricColumn()); // an extra colum that can display additional metric reported by the benchmarks
-
-            UnionRule = ConfigUnionRule.AlwaysUseLocal; // global config can be overwritten with local (the one set via [ConfigAttribute])
         }
 
         protected virtual Job GetJobDefinition()
             => Job.Default
                 .WithWarmupCount(1) // ML.NET benchmarks are typically CPU-heavy benchmarks, 1 warmup is usually enough
-                .WithMaxIterationCount(20);
+                .WithMaxIterationCount(20)
+                .AsDefault(); // this way we tell BDN that it's a default config which can be overwritten
 
         /// <summary>
         /// we need our own toolchain because MSBuild by default does not copy recursive native dependencies to the output
         /// </summary>
-        private IToolchain CreateToolchain()
+        protected IToolchain CreateToolchain()
```

Comment:
Why does this and `GetBuildConfigurationName` need to be `protected`?

Reply:
good catch, solved